frontend/src/concepts/analyticsTracking/initSegment.tsx

```diff
@@ -1,74 +1,27 @@
+import { AnalyticsBrowser } from '@segment/analytics-next';
+
 export const initSegment = async (props: {
   segmentKey: string;
   enabled: boolean;
 }): Promise<void> => {
   const { segmentKey, enabled } = props;
-  const analytics = (window.analytics = window.analytics || []);
-  if (analytics.initialize) {
+  if (!enabled) {
+    window.analytics = window.analytics || [];
```

Comment:
This causes an exception to be thrown when a segment key is provided but `disableTracking=true`. `window.analytics = []` causes the exception in `segmentIOUtils`.
```
TypeError: window.analytics.identify is not a function
    at fireTrackingEvent (segmentIOUtils.tsx:48:26)
```

Reply:
Good catch